convert2rhel/systeminfo.py

```diff
@@ -155,6 +157,31 @@ def _get_system_version(self):
         self.logger.info("%-20s %d.%d" % ("OS version:", version.major, version.minor))
         return version
 
+    def _get_system_distribution_id(self, system_release_content=None):
```

Comment:
Personally I don't think anyone would need this "id" when collecting the breadcrumbs for statistics. "Final" or "Core" doesn't even resemble any kind of identification. In RHEL the parentheses contain a codename, e.g. Ootpa. CentOS Stream has no parenthesis the same as Oracle Linux ("CentOS Stream release 8").
For that reason, I'd remove this function together with publishing this id in the breadcrumbs.

Reply:
This is mostly relevant for `target_os` rather than the `source_os`, indeed, as the codename in RHEL is more important on statistics, I would say we could leave this as it is right now as it won't affect anything or anyone if `source_os` has an extra `id` there (also, making it more “complete”). 

Maybe I misunderstood @tabowling example when we were talking about this feature, probably he wanted the `id` only for the `target_os`, but honestly, I don't see a problem with both of them having the id... Most of the time I think this would be empty for `source_os`.